UPGRADE-7.3.md

```diff
@@ -8,6 +8,11 @@ Read more about this in the [Symfony documentation](https://symfony.com/doc/7.3/
 
 If you're upgrading from a version below 7.1, follow the [7.2 upgrade guide](UPGRADE-7.2.md) first.
 
+Console
+-------
+
+ * Omitting parameter types in callables configured via `Command::setCode` method is deprecated
```

Comment:
You can add before/after like [here](https://github.com/symfony/symfony/blob/55b2ea0087c3337eba67c968115859a035b9199f/UPGRADE-7.2.md?plain=1#L57C1-L77C1).

*Before*
```php
$command->setCode(function ($input, $output) {
    // Your command
});
```

*After*
```php
use Symfony\Component\Console\Input\InputInterface;
use Symfony\Component\Console\Output\OutputInterface;

$command->setCode(function ($input, $output) {
    // Your command
});```